dts/bindings/espi/gpio-espi.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2018, Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO eSPI parent node
+
+compatible: "gpio-espi"
+
+description: GPIOs required for eSPI handshake
+properties:
+   pwrg-gpios:
+      type: phandle-array
+      required: true
+      description: Board GPIO used to detect power rails are stable.
```

Comment:
Document whether this is an input or an output of the device (I probably guessed wrong in my comment at the configuration call where I assumed it was a processor output acting like a switch) and its nominal active level.